src/types/feeBreakdown.ts

```diff
@@ -21,10 +23,14 @@ export type ReductionOrExemption =
 export interface PassportFeeFields {
   "application.fee.calculated": number;
   "application.fee.payable": number;
-  "application.fee.payable.includesVAT": boolean;
   "application.fee.reduction.alternative": boolean;
   "application.fee.reduction.parishCouncil": boolean;
   "application.fee.reduction.sports": boolean;
   "application.fee.exemption.disability": boolean;
   "application.fee.exemption.resubmission": boolean;
+  "application.fee.serviceCharge"?: number;
+  "application.fee.serviceCharge.VAT"?: number;
+  "application.fee.fastTrack"?: number;
+  "application.fee.fastTrack.VAT"?: number;
+  [key: `${string}.VAT`]: number | undefined;
```

Comment:
```suggestion
  "application.fee.serviceCharge": number;
  "application.fee.serviceCharge.VAT": number;
  "application.fee.fastTrack": number;
  "application.fee.fastTrack.VAT": number;
  [key: `${string}.VAT`]: number;
```

I think this is actually correct as the Zod schema is converting these to 0 - `feeSchema.optional().default(0)`

Reply:
This was my original instinct also - but when required, existing tests "inputs" will error that there are missing fields - eg 
![Screenshot from 2025-04-02 18-02-22](https://github.com/user-attachments/assets/c0a8d2e3-867b-40d1-8a55-bfb0ff8c27fc)

Therefore, I think optional will ensure that the fee breakdown doesn't break for existing invite to pay sessions (passport fields already written to `payment_requests`, but actual page/fee breakdown not going to be opened/displayed for up to 28 days) when this is merged?